MyHashMap.java

```diff
@@ -0,0 +1,83 @@
+// Time Complexity :  O(1) amortized. worst case would be O(n)
+// Space Complexity : O(n)
+// Did this code successfully run on Leetcode : yes
+// Any problem you faced while coding this :  on line 66, was comparing 'prev.next.val != key' and spent some time 
```

Comment:
It's good that you are paying attention to such details. Whiteboarding is the solution and keep practicing, you are doing great!